### DISCUSSION TITLE
Does version 1.0 support Chinese text recognition training? Does it only need to modify dicts?

### DISCUSSION THREAD

**Discussion by htaoruan** [open-mmlab/mmocr#1570] [General]
Does it only need to modify dicts?
chinese_english_digits.txt

**Comment by gaotongxiao** [comment]
Yes, the only thing needed is to change the dictionary path.

**Reply by htaoruan** [reply]
OK  Thank you。Is this also true for version 0.6.3？

**Reply by gaotongxiao** [reply]
In 0.6.3, you would have to manually update the decoder's output channel and probably the loss's input channel, depending on the model architecture you are working with.

**Reply by htaoruan** [reply]
Can you please give us an example? For example, using the classic crnn network

**Reply by htaoruan** [reply]
thanks

**Reply by mashiguo** [reply]
Have you solved the problem of training Chinese？
